ipa_room_exploration/common/src/convex_sensor_placement_explorator.cpp

```diff
@@ -296,27 +277,9 @@ void convexSPPExplorator::getExplorationPath(const cv::Mat& room_map, std::vecto
 			else // point not in the right range to be inside the fov
 				V.at<uchar>(neighbor-cell_centers.begin(), pose-candidate_sensing_poses.begin()) = 0;
 		}
-//		cv::imshow("observable cells", black_map);
-//		cv::waitKey();
 	}
 	std::cout << "number of optimization variables: " << W.size() << std::endl;
 
-//	testing
```

Comment:
Please keep this block, otherwise we need to program this again when we need it for debugging.